packages/lib/next-auth/auth-options.ts

```diff
@@ -193,7 +193,15 @@ export const NEXT_AUTH_OPTIONS: AuthOptions = {
       idToken: true,
       allowDangerousEmailAccountLinking: true,
 
-      profile(profile) {
+      profile(profile, tokens) {
+        if (tokens && 'refresh_expires_in' in tokens) {
+          delete tokens.refresh_expires_in;
+        }
+
+        if (tokens && 'not-before-policy' in tokens) {
+          delete tokens['not-before-policy'];
+        }
```

Comment:
`delete` makes me sad, it'd be better to just deconstruct or validate the items we do need and work backwards from that.

I noticed that this also doesn't remove the scarier oidc code.